docs/Common/Structs/index.md

```diff
@@ -540,6 +584,92 @@
 |ventilationMode|VentilationMode|false|array: true <br> minsize: 1 <br> maxsize: 100|A set of all ventilation modes that are controllable|
+|heatedSteeringWheelAvailable|Boolean|false|| Availability of the control (enable/disable) of heated Steering Wheel. <br> True: Available, False: Not Available, Not present: Not Available.|
+|heatedWindshieldAvailable|Boolean|false|| Availability of the control (enable/disable) of heated Windshield. <br> True: Available, False: Not Available, Not present: Not Available.|
+|heatedRearWindowAvailable|Boolean|false|| Availability of the control (enable/disable) of heated Rear Window. <br> True: Available, False: Not Available, Not present: Not Available.|
+|heatedMirrorsAvailable|Boolean|false|| Availability of the control (enable/disable) of heated Mirrors. <br> True: Available, False: Not Available, Not present: Not Available.|  
+
+### AudioControlCapabilities
+
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:----------|
+|moduleName|String|true|maxlength="100"|The short friendly name of the light control module. <br> It should not be used to identify a module by mobile application.|
+|sourceAvailable|Boolean|false||Availability of the control of audio source.|
+|volumeAvailable|Boolean|false||Availability of the control of audio volume.|
+|equalizerAvailable|Boolean|false||Availability of the control of Equalizer Settings.|
+|equalizerMaxChannelId|Integer|false|minvalue="1" <br> maxvalue="100"|Must be included if equalizerAvailable=true, and assume all IDs starting from 1 to this value are valid.|
+
+### EqualizerSettings
+
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:----------|
+|channelId|Integer|true|minvalue="1" <br> maxvalue="100"|Defines the each Equalizer channel settings.|
+|channelName|String|false|maxlength="50"|Read-only channel / frequency name (e.i. "Treble, Midrange, Bass" or "125 Hz")|
+|channelSetting|Integer|true|minvalue="0" <br> maxvalue="100"|Reflects the setting, from 0%-100%.|
+
+### AudioControlData
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:----------|
+|source|Common.PrimaryAudioSource|false||In a getter response or a notification, it is the current primary audio source of the system. <br> In a setter request, it is the target audio source that the system shall switch to. <br> If the value is MOBILE_APP, the system shall switch to the mobile media app that issues the setter RPC.|
+|keepContext|Boolean|false||This parameter shall not be present in any getter responses or notifications. <br> This parameter is optional in a setter request. The default value is false. <br> If it is true, the system not only changes the audio source but also brings the default infotainment system UI associated with the audio source to foreground and set the application to background. <br> If it is false, the system changes the audio source, but keeps the current application's context.|
```

Comment:
<br> If it is `false`, the system not only changes the audio source but also brings the default infotainment system UI associated with the audio source to foreground and set the application to background. <br> If it is `true`, the system changes the audio source, but keeps the current application's context.|

Reply:
@yang1070, thank you.  
The description of the `keepContext` param  
provided in the proposal [0099-new-remote-control-modules-and-parameters.md](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0099-new-remote-control-modules-and-parameters.md)  
section **New AUDIO data types** says:  
```xml  
    <param name="keepContext" type="Boolean" mandatory="false">
      <description>
      This parameter shall not be present in any getter responses or notifications.
      This parameter is optional in a setter request. The default value is false.
      If it is true, the system not only changes the audio source but also brings the default infotainment system UI associated with the audio source to foreground and set the application to background.
      If it is false, the system changes the audio source, but keeps the current application's context.
      </description>  
```
@yang1070, please confirm updating the description according to your comment above.